lib/membrane_rtc_engine/tees/audio_tee.ex

```diff
@@ -0,0 +1,177 @@
+defmodule Membrane.RTC.Engine.AudioTee do
```

Comment:
Why do we need seperate tee for audio? Isn't audio track the same case as for non-simulcast video track?